ddsrouter_core/test/blackbox/ddsrouter_core/dds/InitialPeers/DDSTestInitialPeers.cpp

```diff
@@ -0,0 +1,481 @@
+// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
I also feel like this should be merged with `DDSTestWAN.cpp`, and if not, the latter should change its name to stay consistent with the change of participant names.